pandas/tests/indexes/test_base.py

```diff
@@ -217,11 +205,9 @@ def __init__(self, array):
             def __array__(self, dtype=None):
                 return self.array
 
-        for array in [np.arange(5), np.array(['a', 'b', 'c']),
-                      date_range('2000-01-01', periods=3).values]:
-            expected = pd.Index(array)
-            result = pd.Index(ArrayLike(array))
-            tm.assert_index_equal(result, expected)
+        expected = pd.Index(array)
```

Comment:
can you audit for whether we use ``pd.Index`` (and similar) or ``Index`` (we are not generally consistent), like to be consistent within in a module

Reply:
```sh
grep " Index(" -r pandas/tests --include="*.py" | wc -l
     891
grep " pd.Index(" -r pandas/tests --include="*.py" | wc -l
     264
```

May be a more robust way of doing it but assuming directionally accurate first is more widely used. Particular to this module I see numbers of 175 and 48, respectively.

I personally prefer `pd.Index` for explicitness but am fine to change to `Index` for module-consistency. lmk